megaavr/extras/testCases/disableAllPeripherals_testCase/README.md

```diff
@@ -0,0 +1,3 @@
+### Note a couple of things before using this test case
```

Comment:
I think a brief description of what the expected results for this test case are would be good to add. Otherwise, this looks good.